backend/src/main/java/WebInterfaceImpl.java

```diff
@@ -13,16 +13,17 @@
  * Created by danie on 2017-09-28.
  */
 public class WebInterfaceImpl implements WebInterface {
-    Persistance dbConnector = new PostgresSQLConnector();
+    DBInterface dbConnector = new PostgresSQLConnector();
+    GenericRepository<Customer> customerGenericRepository = new GenericRepository<Customer>(Customer.class,dbConnector);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)